docs/setup_guides/azure.rst

```diff
@@ -0,0 +1,249 @@
+Installing Pangeo on Azure
+--------------------------
+
+This guide takes you through the steps necessary to install Pangeo on Microsoft's Azure Cloud Platform.
+We'll make use of Azure's Kubernetes as a Service offering, called AKS (Azure Kubernetes Service),
+for installing Pangeo on Azure.
+Documentation on AKS can be found here: https://docs.microsoft.com/en-gb/azure/aks/.
+
+.. Note::
+  This guide lays out only the fundamental steps required to install Pangeo on Azure AKS.
+  Further work, for example to secure your cluster, is highly advised but not directly
+  covered here.
+
+
+Step One: Build Kubernetes service
+==================================
+
+The first step to installing Pangeo on Azure is to set up a Kubernetes service
+that can be used to run Pangeo. This can be done either by using the web interface
+or by using the Azure commandline interface (CLI). These are both practical options,
+so we'll cover each one in turn.
+
+Using the web interface
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To use the Azure web interface you must first have a Microsoft account that you
+can use to log into the Azure web interface. If you have an existing Microsoft
+account (for example, a hotmail.com or outlook.com email address) then you can use
+that, or you can create a new account.
+
+Once you have logged into the Azure web interface, navigate to Kubernetes services
+and click the blue Add logo in the top left. This will display the Create Kubernetes cluster
+wizard.
+
+Work through the wizard customising the Kubernetes service to be created as you
+see necessary (though the defaults are largely reasonable). In the last step before
+the cluster is created a validation process is run, ensuring that the customisations you have
+made will produce a working cluster. At this step you can also download a
+template file to make reproducing or automating cluster creation simpler in the future.
+
+Autoscaling
```

Comment:
Should we mention that traditional autoscaling is not possible with the GUI and needs to be done by the CLI?